chef/cookbooks/provisioner/templates/default/crowbar_join.suse.sh.erb

```diff
@@ -133,7 +133,7 @@ sync_time() {
     local tries_left=120
 
 <% if @target_platform_version.to_f < 12.0 %>
-    SNTP_OPTS="-P no -r"
+    SNTP_OPTS="-S"
```

Comment:
Is there any reason for using ```-S``` on SLE11 and ```-s``` on SLE12 (and newer). AFAIK both release are using the same ntp packages now.